packages/api-explorer/package.json

```diff
@@ -19,13 +19,13 @@
   "scripts": {
     "test": "jest",
     "build": "tsc && webpack --config webpack.prod.config.js",
-    "develop": "webpack-dev-server --https --disable-host-check --config webpack.dev.config.js",
+    "develop": "webpack-dev-server --host=0.0.0.0 --disable-host-check --config webpack.dev.config.js",
```

Comment:
Allow use in proxied development environments that don't forward SSL and need host bound to 0.0.0.0